src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -148,28 +159,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             // Adds Brotli and Gzip compressing
             app.UseResponseCompression();
 
-            // Enable middleware to serve generated Swagger as a JSON endpoint.
-            app.UseSwagger();
-
-            // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.), 
-            // specifying the Swagger JSON endpoint.
-            app.UseSwaggerUI(c =>
-            {
-                c.SwaggerEndpoint("/swagger/v1/swagger.json", "Data API V1");
-                c.RoutePrefix = "docs";
-            });
-
-            if (!env.IsDevelopment())
-            {
-                app.UseHttpsRedirection();
```

Comment:
This seems to have been removed.

Reply:
It was redundant - the same thing is called above.